libs/blocks/global-navigation/global-navigation.js

```diff
@@ -4,12 +4,67 @@ import {
   getConfig,
   getMetadata,
   loadIms,
+  loadStyle,
+  loadLana,
   decorateLinks,
   loadScript,
   getGnavSource,
+  getFederatedUrl,
   getFedsPlaceholderConfig,
 } from '../../utils/utils.js';
-import {
+
+(async () => {
+  const { miloLibs, codeRoot, theme } = getConfig();
+  const url = `${miloLibs || codeRoot}/blocks/global-navigation/`;
+  const loadStylePromise = (u) => new Promise((resolve, reject) => {
+    loadStyle(u, (e) => {
+      if (e === 'error') return reject(u);
+      return resolve();
+    });
+  });
+  try {
+    await loadStylePromise(`${url}base.css`);
+    if (theme === 'dark') await loadStylePromise(`${url}dark-nav.css`);
```

Comment:
Does the css loading need to be `await`ed?  If so - probably better to parallel load these
```suggestion
    await Promise.all([
      loadStylePromise(`${url}base.css`),
      theme === 'dark' ? loadStylePromise(`${url}dark-nav.css`) : true,
    ]);
```

Reply:
`dark-nav.css` must load after base.css. If we load it in parallel, then if `dark-nav.css` loads first, the dark mode gnav will break (this is for a specific standalone consumer). In Milo consumers we don't have dark mode.